test/ng/directive/inputSpec.js

```diff
@@ -3017,10 +3629,10 @@ describe('input', function() {
         ['scheme://example.com/path?query#~`!@#$%^&*-_=+|\\;:\'",.?/()[]{}<>', true],
 
         // Validating miscellaneous
-        ['scheme://☺.�.⌘.➡/䨹', true],
-        ['scheme://مثال.إخ�بار', true],
+        ['scheme://☺.?⌘.➡/䨹', true],
+        ['scheme://مثال.إخ??ار', true],
         ['scheme://例子.测试', true],
-        ['scheme://उदाहरण.�रीक्षा', true],
+        ['scheme://उदाहरण.???ीक्षा', true],
```

Comment:
Hmm, I wonder if this change is right? It looks like previously the strings were UTF-16?